configs/templates/jedi-ufs-all-linux-gcc/spack.yaml

```diff
@@ -0,0 +1,26 @@
+spack:
```

Comment:
I would create one jedi-ufs-all template and add to the instructions/CI tests commands to remove certain spack specs that don't build on one or the other system.

Then, remove the individual jedi-ufs-all-OS-COMPILER files.

Reply:
That makes sense. One default list with a little manual intervention is better than carrying around 4 or5 files.